VocaDbWeb/Views/SongList/Featured.cshtml

```diff
@@ -52,10 +52,19 @@
 		$("#createLink").button({ icons: { primary: 'ui-icon-plusthick' } });
 		$("#importLink").button({ icons: { primary: 'ui-icon-plusthick' } });
 		$("#tabs").tabs();
+		
+		var cultureCode = '@UICulture';
 
-		var urlMapper = new vdb.UrlMapper('@RootPath');
+		var languageSelection = '@UserContext.LanguagePreference';
+		var rootPath = '@RootPath';
+		var urlMapper = new vdb.UrlMapper(rootPath);
+		var repoFactory = new vdb.repositories.RepositoryFactory(urlMapper, @LanguagePreferenceInt, @UserContext.LoggedUserId);
+		var songListRepo = repoFactory.songListRepository();
+		var resourceRepo = repoFactory.resourceRepository();
+		var tagRepo = repoFactory.tagRepository();
+		var languageSelection = '';

```

Comment:
Why are you overriding languageSelection here?

Reply:
I guess it must be a mistake. Thanks.